app/src/components/Layout.tsx

```diff
@@ -10,11 +10,15 @@ const Layout = ({ children }: Props): ReactElement => {
 
   return (
     <div className="container">
```

Comment:
nit: we can remove this too, along with its custom CSS 

```suggestion
    <div>
```